tests/utils/admincli/vmgroupmgmt.go

```diff
@@ -93,13 +121,38 @@ func ConfigInit(ip string) (string, error) {
 	}
 
 	log.Printf("Initializing the SingleNode Config DB on esx [%s] \n", ip)
-	return ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+	out, err := ssh.InvokeCommand(ip, admincli.InitLocalConfigDb)
+
+	if err != nil {
+		return out, err
+	}
+
+	// verify the DB init
+	if GetDBmode(ip) != admincli.DBSingleNode {
```

Comment:
This logic seems not exactly consistent with what we are doing in above CreateVmGroup and DeleteVmGroup. If we follow the same style, the logic here should be something like this:

```golang
if GetDBmode(ip) == admincli.DBSingleNode {
    return out, nil
}

msg := ...
return msg, fmt.Errorf(msg)
```
Not a big concern, but I'd like to know if there are any specific reason for this difference.